alternator/executor.cc

```diff
@@ -2786,7 +2786,7 @@ static void verify_all_are_used(const rjson::value* field,
         if (!used.contains(rjson::to_string(it->name))) {
             throw api_error::validation(
                 format("{} has spurious '{}', not used in {}",
-                    field_name, it->name.GetString(), operation));
+                    field_name, field_name, operation));
```

Comment:
You forgot to fix this bug you introduced in the previous version of your patch :-( It shouldn't be field_name twice.

Reply:
I didn't I removed my new variable, but forgot to remove GetString here, anyway fixed now